### ISSUE TITLE
Admin UI underpowered in 2.0

### ISSUE THREAD

**Issue by wojons** [rethinkdb/rethinkdb#4112]
So i am really happy that the cluster can be managed with reql, but i am also finding myself having to write reql for some simple things i use to do in the dashboard that i know had to do by hand. Is there anyway to re-ad the features back in to the dahsboard and just have them make the calls to the database.

**Comment by deontologician** [comment]
What features are you missing? The only one I'm aware of currently is the
ability to rename databases

**Comment by mglukhovsky** [comment]
@deontologician: we also removed MDC features because of the admin re-write.

@wojons: which features are you referring to?

**Comment by wojons** [comment]
@mglukhovsky 

dont get me wrong i like the abilty to submit queries to manage my cluster and there are some parts of my code that i will do it but  i did not like having to learn it to just get some things going for example i use to be able to click in the ui to rebalance i use to add replica in data centers now tags in the ui.

**Comment by danielmewes** [comment]
We decided to omit server tags in the 2.0 web UI rewrite because they are a rather advanced feature.
I agree that in the long run we'll want to get those features (and more) back into the UI.

For rebalancing I opened https://github.com/rethinkdb/rethinkdb/issues/4120 .

**Comment by mglukhovsky** [comment]
@wojons: indeed. The semantics of how MDC worked changed when we added rethinkdb-admin (thanks to the introduction of server tags), so we decided to ship without a redesigned UI, and keep it simple. We should definitely add those back when we have time.

**Comment by thelinuxlich** [comment]
Also, I think the tags feature deserves better documentation

**Comment by danielmewes** [comment]
@thelinuxlich I assume you've seen the "Advanced configuration" http://rethinkdb.com/docs/sharding-and-replication/ ? I agree that we could (and should) write a lot more about it, but are there specific aspects that you think are absent from our current documentation in particular?

**Comment by mglukhovsky** [comment]
@thelinuxlich: I opened rethinkdb/docs#733 to expand our docs on server tags.

**Comment by wojons** [comment]
Yeah i understand i think if your going to have a large gab between 2.0 and what ever falls next there should be an upgrade to the dashboard because beforei t was super easy to shard a table no reql need to know. just click click click.
